### PR TITLE
Feature/media query adjustments

### PR DESCRIPTION
**Pull request by pcmueller**
## Type of Change
- [x] JSX
- [x] HTML
- [ ] JavaScript
- [x] CSS
- [ ] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [x] Refactor/Fix Bugs

## Please describe the change made
CSS/media query refactoring

## Please give a short description of the changes that were made below

- This branch has been going since last week, so it's maaaany lines of code... but basically all CSS / media query refactoring. 
Lots of work just getting everything sized right, especially keeping the header, navbar, and main elements (form, current trip, etc.) proportionate on all screen sizes.  I've merged and modified all of the latest design changes (light/dark mode, toaster popups, etc), so this does incorporate all of our up-to-date code, and shouldn't lead to many conflicts.  

- I moved the themeToggle element (using the CheckBox, CheckBoxLabel, CheckBoxWrapper components) from `App.js` down to `NavBar.js`, where it is rendered inside a container div alongside the hamburger dropdown, in order to keep these two together across all media query sizes.  The 'switchTheme' prop still lives in `App.js` (where the theme is still manipulated), the prop is just drilled down into `NavBar.js` now, so that it can be toggled there by the user.

Components with completed media queries / display improvements:
- Header
- NavBar (including themeToggle and hamburger components)
- Form (including inputs and old dropdowns)
- MainView
- TripETA
- CurrentTrip

To-do list:
- AboutUs (improved but still needs work)
- AddContact ( " )
- AddTime
- Alert

## How has this been tested?
- [x] In Browser
- [ ] In Console
- [ ] Cypress
- [ ] Postman

### CODE REVIEW

**Review by carolineshowscode — APPROVED** [review]
Looks good!

File: src/Components/AboutUs/AboutUs.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ function AboutUs({ user }) {
 
   return (
     <main className='about-us-page'>
-      <NavBar nameToggle={false} user={user.firstName}/>
+      <NavBar nameToggle={false} user={user.firstName} switchTheme={switchTheme} />
```
Comment (bhohnco) [inline src/Components/AboutUs/AboutUs.js:56]: Nice work here Pete!

File: src/Components/App/_app.scss [hunk 2]
```diff
@@ -41,116 +40,3 @@ body,
...
-    height: 25%;
-  }
-
-}
-
-@media screen and (max-width: 930px) {
-  .mm-popup__box {
-    width: 60%;
-    height: 25%;
-  }
```
Comment (bhohnco) [inline src/Components/App/_app.scss:127]: Good call removing all of this

File: src/theme.js [hunk 3]
```diff
@@ -1,22 +1,27 @@
-import {createGlobalStyle } from "styled-components"
-import styled from "styled-components";
+import { createGlobalStyle } from 'styled-components'
+import styled from 'styled-components';
 
 export const CheckBoxWrapper = styled.div`
   position: relative;
+  display: flex;
```
Comment (bhohnco) [inline src/theme.js:6]: Nice work making this responsive

**Review by bhohnco — APPROVED** [review]

**Pull request merged**